src/services/videoconference/index.js

```diff
@@ -0,0 +1,537 @@
+const {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File has too many classes (3). Maximum allowed is 1.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)